index.html

```diff
@@ -233,6 +234,16 @@ <h3>MathML Element Mappings</h3>
 		    <span class="subrole">AXSubrole: <code>TBD</code></span>
 		  </td>
 		</tr>
+                <tr id="el-math">
+                  <th>
+                    <a data-cite="html/embedded-content-other.html#mathml">`math`</a>
+                  </th>
+                  <td class="ia2"><div class="general">Use WAI-ARIA mapping</div></td>
+                  <td class="uia"><div class="general">Use WAI-ARIA mapping</div></td>
+                  <td class="atk"><div class="general">Use WAI-ARIA mapping</div></td>
+                  <td class="ax"><div class="general">Use WAI-ARIA mapping</div></td>
+                  <td class="aria"><a class="core-mapping" href="#role-map-math">`math`</a> role</td>
```

Comment:
it seems to be a local link? but should refer to ARIA spec?

Reply:
yes, this was a local link from HTML AAM... I'll check that.

Reply:
OK, it's actually referring to the Core AAM. I copied the respec config so that it still works. However the local ref for the mapping of the math role is currently broken (this is already the case in the HTML AAM spec).

Reply:
got it, I was testing that locally. Just to confirm, no other changes required to make it working in MathML AAM?

Reply:
we can fix the broken link but this has to be done on the Core AAM side first, as apparently there is currently no anchor we can point to...

(by "I copied the respec config so that it still works." I mean I pushed https://github.com/w3c/mathml-aam/pull/21/commits/13068314b48aca45135abc4b29e4e326bb7afc2f)

Reply:
ok, is there anything else to fix here before merging?

Reply:
I think we can merge this. I'll open an issue on the Core AAM side.

Reply:
So the reference is correct https://github.com/w3c/core-aam/blob/e7776e1dddca66bb342b5e84446c5444328cc66f/index.html#L1238

But there seems to be a bug in general with how local links interact with the generated `<details>` elements.